### ISSUE TITLE
Missing inaccessible icon

### ISSUE THREAD

**Issue by pdpinch** [mitodl/hugo-course-publisher#132]
OCW uses a [transparent gif to indicate inaccessible content](https://ocw.mit.edu/images/inacessible.gif). It's invisible to sighted users, but its ALT text is read by a screen reader. 

We need to add the image to static assets. 

### Steps to Reproduce

The page at https://master--ocw-next.netlify.app/courses/12-001-introduction-to-geology-fall-2013/sections/lecture-notes-and-slides/ is returning a 404 for https://master--ocw-next.netlify.app/images/inacessible.gif
